cpu/stm32_common/stm32_mem_lengths.mk

```diff
@@ -190,7 +190,13 @@ ifeq ($(STM32_TYPE), F)
   endif
 else ifeq ($(STM32_TYPE), L)
   ifeq ($(STM32_FAMILY), 0)
-    ifneq (, $(filter $(STM32_MODEL2), 1 2))
+		ifeq ($(STM32_MODEL2), 1)
+			ifeq ($(STM32_MODEL3), 0)
+				ifeq ($(STM32_ROMSIZE), 6)
+					RAM_LEN = 8K
+				endif
+			endif
+		else ifneq (, $(filter $(STM32_MODEL2), 1 2))
```

Comment:
Please use two spaces for indentation to be consistent with the rest of the file.